Controller/Api/GalleryController.php

```diff
@@ -343,7 +343,7 @@ protected function handleWriteGalleryhasmedia(GalleryInterface $gallery, MediaIn
             'csrf_protection' => false,
         ));
 
-        $form->bind($request);
+        $form->handleRequest($request);
```

Comment:
This is "boy scout rule" yes is not related to PR but is improving the code not damage it!